### ISSUE TITLE
Sign off for Site Map

### ISSUE THREAD

**Issue by EChesters** [empowerhack/DrawMyLife-Website#2]
## What

_This is UX only at this point._
The mobile and desktop wireframes need signing off in terms of _design_ and _content_.
## How

Need @rocioleon's input on design and content. Possibly @CathMollie to input on content.
## To Do
- Design requirements

~[DML-website.pdf](https://github.com/empowerhack/DrawMyLife-Website/files/532077/DML-website.pdf)~ Old wireframes

**Comment by rocioleon** [comment]
Hi,
A few things to consider and discuss at some point:

Rg CONTENT: what is main content and supporting content. Also we need to write down TEXT using the tone of voice we agreed last 15Oct. and select images that you are thinking to show.

Rg NAV.: use the correct labels so we really know what are going to be looking at.

Rg page 3. We are not using any forms in this website but contact us? well I don't know if the contact us page will exists

Rg page 4. what are those orange shapes with the letter 'A' in it?

I know that this website is not high priority but let me know if you need to discuss this with me. Otherwise I will focus on the DU-campaign

**Comment by tanyapowell** [comment]
Hey @EChesters - great first draft! Just a couple of questions
- Will we be having a footer?
- Will we be having a team page? Especially as we are all volunteering our time it would be nice to have a humble brag 😏
- Do you have an example of an existing website with something similar to this? I've seen it on another website (can't remember what site) and found it quite jarring 
  ![image](https://cloud.githubusercontent.com/assets/11095605/19626801/9fa253ae-9930-11e6-9fe5-18fa45a39ed6.png)

**Comment by EChesters** [comment]
Okay so given the feedback I decided to ditch the wireframes. It doesn't help to see everything what we need for the website. It's become apparent that that it won't be one page like I first imagined. 

So behold! I've done a site map similar to what @rocioleon did for the DU Campaign site. Feedback from anyone welcome!
![dml-website-sitemap](https://cloud.githubusercontent.com/assets/2605703/19662907/7e8a765e-9a31-11e6-9114-9fff458c5151.png)
The red of EmpowerHack is a question of whether we should include this at all? 

For the questions still relevant:
**Yes, we will have a footer. I don't like them so I leave them till the end to design**
**I have seen that design before but can't remember where - but wireframes are not about design! That was a rookie mistake by me**
**Yes we will also have a team page. At the moment we all have matching photos - @tanyapowell I need to take your photo and try to get it to match!**

**Comment by tanyapowell** [comment]
@EChesters you mean I can't send over a snap 😉

**Comment by EChesters** [comment]
No, we all match :P I have a picture of the background though if you'd like to photoshop yourself in that 😆

**Comment by rocioleon** [comment]
@EChesters wow very good and clear! 
Rg red text EmpowerHack I think we need to include it. I saw a message from Han on slack a couple of days ago where Han was asking to be cc (and Kimi) on any email about partnerships and stuff like this. But, please confirm this with them.

Rg rest, I think you are right, maybe we need to start thinking about the Copy as well to see how much copy we need and images and visuals we need.
Well done!

**Comment by CathMollie** [comment]
@EChesters Great wireframe. 
- In terms of volunteering to be part of the team, I think all recruitment is done through EH so Kimi or Han might have a view on whether we call out to people here as well. All the partnering stuff is fine.
- Is it worth thinking about the different audiences and their goals? While this is beautifully organised I feel like we're hitting one general audience and might be missing an opportunity to make journeys to suit certain purposes. At the same time want to avoid silos. Be very interested to know if others share same views as we didn't have much of a chance to work this through in our Design Day.

Techie - here to find out more about data and interested in working with API to get data out and make cool stuff with it. May be interested in working with us on a joint venture/platform to surface data - a bit like Erehwon! Basically @marizoldi is one of our personas ...

NGO - here to get involved in data collection or view some of the data that has been collected and share it with their org or find out ways to access it for particular purposes e.g. advocacy and research. They'd be interested in knowing with impact/value the data had made so far and whether worth their while to dig deeper. Good opportunity to point to our partners HDX as bigger repository of our data.

Public - less obvious goal, strong emotive interest in data and drawings, and want low-friction ways to help e.g. by sharing. Probably interested in reading about some of the contexts e.g. current conditions in camps/distribution centres. May get inspired by seeing ways in which the data has been used and feel inspired to start something similar.

**Comment by krissy** [comment]
Well done on this sitemap! And :+1: to the feedback so far too.

Scattered thoughts (let me know if any of this doesn't make sense):
- Re empowerhack - yes, this will always be an EmpowerHack baby and need to make that relationship clear, even on main layout I'd say.
- Also need a big link to HDX, e.g. "Find us on the Humanitarian Data Exchange".
- I'm wondering if we need a whole Volunteer section, or if we have it as a section under Meet The Team or Contact Us, and then any CTA would link to that anchor rather. I like @CathMollie's idea of adding a Tech Community section we can link off to the API documentation, but also advertise for volunteers and our GitHub repo to encourage any open source contributions. 

Overall :+1: :dancing_men:

**Comment by CathMollie** [comment]
@krissy @EChesters Hadn't thought about source contributions and totally see what Elizabeth was getting at now, great idea!!

**Comment by EChesters** [comment]
Okay so main points that I'll be adding later to the mind map
- YES to EmpowerHack section and actually include this more on the team page
- YES we will be catering towards different people and their needs which is why for a simple website we have a lot of pages. The more we interview our personas the more we'll be able to change our home page to point them in the right direction. Additionally, each page like volunteer/team/data will be treated as landing pages so we can "skip" the home page when we share the website with different people.
- Add HDX link to the Data page

The volunteer section was aimed at tech people, and when treated as a landing page it is much bigger than the team section. We already have a lot on both of these pages, so we can link them and add same sections on both pages, but keeping them separate I think would be hugely beneficial. 

I'll also speak to Han about how we get her and Kimi involved with the partnerships and if we are contacted we still need to decide how we monitor and take care of that bit. 

Anything I've missed from that list of what's going to change on the map?

**Comment by EChesters** [comment]
Updated site map, with EmpowerHack as a definite (with a link to the EH website) and the HDX CTA added to the Data page.

![dml-website-sitemap](https://cloud.githubusercontent.com/assets/2605703/19831705/8adc6252-9e08-11e6-8d15-32a06b606b26.png)

After this, I'm going to start sketching sections, and then build the pages from the sections rather than a page at a time. This also means whilst I'm starting sketching, it'll be cheap so feedback can be integrated at any time! So send me your thoughts!

**Comment by tanyapowell** [comment]
@EChesters what do you mean `build the pages from the sections rather than a page at a time`?

Also, @CathMollie @rocioleon @krissy are you all happy with this sitemap? Can this issue be closed?

**Comment by EChesters** [comment]
Some of the sections have been reused across different pages. Plus with the idea of OOUX, and atomic design, we can build a section sign that off and then when all sections for a page is done, the epic can be signed off 😁 This means that 1 person can pick up Home page banner, and someone else can do Partners section.

**Comment by CathMollie** [comment]
@EChesters Just been looking at the site map again. Will the section on Data also talk about the drawings? Given that DML has two main collections to share with the world: drawings and visualisations of the data.
